lib/index.js

```diff
@@ -15,8 +15,9 @@ exports.register = (server, options, next) => {
     }
 
     const rate = routeSettings.rate ? routeSettings.rate(request) : options.defaultRate(request);
+    const rateLimitKey = routeSettings.rateLimitKey ? routeSettings.rateLimitKey : options.rateLimitKey;
```

Comment:
I think it'd be more clear if you call the functions here instead of below: 

```js
const rateLimitKey = routeSettings.rateLimitKey ? routeSettings.rateLimitKey(request) : options.rateLimitKey(request);
```